src/styles/globalStyles.css

```diff
@@ -9,3 +9,9 @@
 .MuiTimelineItem-missingOppositeContent:before {
   flex: unset !important;
 }
+
+@media screen and (min-width: 703px) and (max-width: 900px) {
```

Comment:
Adjusting the width of the element with a media query seems like the way to go! But I do try to minimize using the global stylesheet - material UI provides a utility for doing this kind of thing within the component:

https://material-ui.com/components/use-media-query/